From 5e59ef29b7c5a0beeb2d95482389de7949b69785 Mon Sep 17 00:00:00 2001
From: Kavita Kavita <kavita.kavita@oss.qualcomm.com>
Date: Thu, 11 Dec 2025 18:44:41 +0530
Subject: [PATCH 27/29] tests: Enable CONFIG_ENC_ASSOC for hwsim wpa_supplicant

Enable CONFIG_ENC_ASSOC option for wpa_supplicant in hwsim to run EPPKE
related tests.

Signed-off-by: Kavita Kavita <kavita.kavita@oss.qualcomm.com>
---
 tests/hwsim/example-wpa_supplicant.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/hwsim/example-wpa_supplicant.config b/tests/hwsim/example-wpa_supplicant.config
index a43bdc5ce..d3dfd4903 100644
--- a/tests/hwsim/example-wpa_supplicant.config
+++ b/tests/hwsim/example-wpa_supplicant.config
@@ -169,3 +169,4 @@ CONFIG_PASN=y
 CONFIG_NAN_USD=y
 
 CONFIG_IEEE80211BE=y
+CONFIG_ENC_ASSOC=y
-- 
2.43.0

